src/components/pill/pill-test.stories.tsx

```diff
@@ -73,3 +92,29 @@ Default.story = {
     colorVariant: "neutral",
   },
 };
+
+StatusDarkBackground.story = {
+  argTypes: {
+    colorVariant: {
+      options: [
+        "neutral",
+        "negative",
+        "positive",
+        "warning",
+        "information",
+        "neutralWhite",
```

Comment:
**praise**: nice job only including the `neutralWhite` variant for the dark background example 👍🏼